lib/EventRider.js

```diff
@@ -24,9 +24,13 @@ module.exports = class GoogleAnalyticsEventRider {
       }
     );
 
+    let countryCode;
```

Comment:
@rmberrios what's this change for?

Reply:
https://github.com/VoxaAI/voxa-ga/pull/23/files/4ee24ea6b1f8898dc0a4b50205c4c9ecc3d49be4#diff-bf7e5bfd41e52774aa847a8a2521a8b4R33

If you don't define specific country languages on dialogflow, you won't receive a country code